.github/workflows/docker-coding-standards.yml

```diff
@@ -0,0 +1,20 @@
+name: "docker coding standards"
+
+on: ["pull_request", "push"]
+
+jobs:
+  coding-standards:
+    name: "docker coding standards"
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "build the environment"
+        run: "dev/bin/docker-compose build"
+
+      - name: "install dependencies"
+        run: "dev/bin/php composer install"
```

Comment:
do we really need to install dependencies for this job?

Reply:
Yes.

```
 dev/bin/php psalm --shepherd --stats
Could not find any composer autoloaders in /app/src/
Add a --root=[your/project/directory] flag to specify a particular project to run Psalm on.
```

 But is not needed for checking coding standards. I'll update the PR